app/views/draws/_oversub_report.html.erb

```diff
@@ -1,4 +1,5 @@
 <h3>Draw Summary</h3>
+<%= link_to 'Handle Oversubscription', oversub_draw_path(draw), class: 'button' if draw.oversubscribed? %>
```

Comment:
this needs to validate that all groups in an oversubscribed size are locked before proceeding